cognite/client/_api/extractionpipelines.py

```diff
@@ -115,22 +116,26 @@ def list(self, limit: int = 25) -> ExtractionPipelineList:
         return self._list(list_cls=ExtractionPipelineList, resource_cls=ExtractionPipeline, method="GET", limit=limit)
 
     @overload
-    def create(self, extractionPipeline: ExtractionPipeline) -> ExtractionPipeline:
+    def create(self, extraction_pipeline: ExtractionPipeline, **kwargs: Dict[str, Any]) -> ExtractionPipeline:
         ...
 
     @overload
-    def create(self, extractionPipeline: Sequence[ExtractionPipeline]) -> ExtractionPipelineList:
+    def create(
+        self, extraction_pipeline: Sequence[ExtractionPipeline], **kwargs: Dict[str, Any]
+    ) -> ExtractionPipelineList:
         ...
 
     def create(
-        self, extractionPipeline: Union[ExtractionPipeline, Sequence[ExtractionPipeline]]
+        self,
+        extraction_pipeline: Union[ExtractionPipeline, Sequence[ExtractionPipeline]],
```

Comment:
won't this break existing clients since `extraction_pipeline` is now a required argument?

Reply:
Oh, that's true, good catch! Btw, are you okey with these "camelCase depreations"? Or would you rather we `noqa`'d them?

Reply:
Made optional in #29f8078